docs/user_docs/kubeblocks-for-redis/cluster-management/create-and-connect-a-redis-cluster.md

```diff
@@ -109,13 +109,13 @@ KubeBlocks supports creating two types of Redis clusters: Standalone and Replica
 Create a Standalone.
 
 ```bash
-kbcli cluster create redis <clustername>
+kbcli cluster redis --mode standalone <clustername>
 ```
 
 Create a Replication Cluster.
 
 ```bash
-kbcli cluster create redis --mode replication <clustername>
```

Comment:
I prefer to keep `--mode replication`, so that it does not rely on the default value of this command (which may change in the future).